src/Tasks/Microsoft.NET.Build.Tasks/Microsoft.NET.Build.Tasks.csproj

```diff
@@ -84,7 +84,7 @@
   <Target Name="PrepareAdditionalFilesToLayout" BeforeTargets="AssignTargetPaths">
     <PropertyGroup>
       <_NugetBuildTasksPackPath>$(NuGetPackageRoot)nuget.build.tasks.pack\$(NuGetBuildTasksPackVersion)</_NugetBuildTasksPackPath>
-      <_Stage0SdksFolder>$(DOTNET_INSTALL_DIR)\sdk\$(NETCoreSdkVersion)\Sdks</_Stage0SdksFolder>
+      <_Stage0SdksFolder>$(RepoRoot).dotnet\sdk\$(NETCoreSdkVersion)\Sdks</_Stage0SdksFolder>
```

Comment:
>$(RepoRoot).dotnet [](start = 25, length = 18)

Can we not hardocde `.dotnet` here?

Reply:
The only variable without .dotnet was `_` private that I found.

Reply:
Couldn't you call `Path.GetDirectoryName($(DotNetTool))`?

Reply:
OK

Reply:
Actually... it is somewhat more complicated :(. Remember how we discussed how to find the `dotnet.exe`? We concluded that we don't need the path since `Exec` resolves it on `PATH`. So the value of `DotNetTool` may be just `dotnet.exe`. In your case it won't though, since you always install it to the repo. Hmm... maybe keep the hardcoded `.dotnet` and file an issue to follow up.

Reply:
Reverted, this wasn't the problem. It will just not copy some files in the VS build, but doesn't block the VS flow, the other parts of this fixed it without this change. This didn't do what I thought it did because it defpes on NetCoreSdkVersion.